Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:bionic
+
+RUN useradd -m retdec
+WORKDIR /home/retdec
+ENV HOME /home/retdec
+
+RUN apt-get -y update && \
+	DEBIAN_FRONTEND=noninteractive apt-get install -y build-essential git bc graphviz upx cmake python zlib1g-dev flex bison libtinfo-dev autoconf pkg-config m4 libtool wget
+
+USER retdec
+RUN git clone --recursive https://github.com/avast-tl/retdec && \
```

Comment:
I would prefer to use `COPY`. 
Could it be optional somehow? Maybe two Dockerfiles: one for local changes and one from upstream master?

Reply:
That's what I suggested.

Since it's not my use case, I'll let someone else create a "development" Dockerfile that they like.